picard/tagger.py

```diff
@@ -287,9 +301,9 @@ def _run_init(self):
                     self.add_directory(decode_filename(file))
                 else:
                     files.append(decode_filename(file))
-            if files:
-                self.add_files(files)
             del self._cmdline_files
+            if files:
+                self.add_files([files])
```

Comment:
I don't get why you need to pass list of lists of filenames, but i didn't check the code in details yet. I'm worried about such change concerning functions like `tagger.add_files()` that may be used in plugins (we really need to clarify plugins API in future versions).